extensions/random-password-generator/src/util.ts

```diff
@@ -31,16 +32,12 @@ const setNumber = (isNumeric: boolean) => {
   return "";
 };
 
-const getRandomInteger = (min: number, max: number) => {
-  return Math.floor(Math.random() * (max - min + 1)) + min;
-};
-
 const passwordCharacters = (): string => {
   const characterList = characters;
   let password = "";
   if (characterList.length > 0) {
     for (let i = 0; i < passwordLength; i++) {
-      password += characterList[getRandomInteger(0, characterList.length - 1)];
+      password += characterList[randomInt(characterList.length)];
```

Comment:
The `-1` is not needed when using `randomInt` because the upper-bound of the function, `max`, is exclusive.